test/unit/fixture/reverse_with_layer_filtering_non_coarse_subset.js

```diff
@@ -0,0 +1,41 @@
+var vs = require('../../../query/reverse_defaults');
+
+module.exports = {
+  'query': {
+    'bool': {
+      'filter': [
+        {
+          'geo_distance': {
+            'distance': '500km',
```

Comment:
I can't speak to why the test says `500km` but it makes sense to update to something much smaller.  This is just a unit test, they don't actually query ES.  

This sanitizer can probably be cleaned up since `boundary:circle:radius:coarse` doesn't mean anything anymore:  

https://github.com/pelias/api/blob/master/sanitizer/_geo_reverse.js#L33-L37